### PR TITLE
Orion 2.0

### PR DESCRIPTION
**Pull request by ImplicitDeny**
Fixing the legacy version of Orion before adding new features.

### CODE REVIEW

File: init.sqf [hunk 1]
```diff
@@ -1,21 +1,16 @@
+// Enregistrement des commandes de requête Zeus
+["zeus", {(_this select 0) spawn GSRI_fnc_handleRequest}, "all"] call CBA_fnc_registerChatCommand;
+
 //EOS SYSTEM
```
Comment (ArwynFr) [inline init.sqf]: What is this ?

Comment (ImplicitDeny) [reply init.sqf]: Used to register the `zeus `command in IG chat. So you can write `#zeus <command>` and the handler will take care of it.

**Comment by ImplicitDeny** [comment]
> What are /scripts/document1.sqf and /scripts/document2.sqf ?
They are used for some of the predefined side missions I think. I might be able to change the location to a proper file, with a better way to handle this, but for now it works as intended. Gonna try to get this clean and tidy.

File: init.sqf (hunk 1 above)
Comment (ArwynFr) [reply init.sqf]: My point was line 4 : EOS SYSTEM

Comment (ImplicitDeny) [reply init.sqf]: Out-of-the-box occupation system for red-colored grid zones. EOS = Ennemy Occupation System.

File: initPlayerLocal.sqf [hunk 2]
```diff
@@ -1,4 +1,5 @@
 //Reserved Slot
+//todo zeus ?
```
Comment (ArwynFr) [inline initPlayerLocal.sqf]: Should be above ;)

File: initPlayerLocal.sqf [hunk 3]
```diff
@@ -1,4 +1,5 @@
...
 if (str(player) in ["p7", "p7_1", "p7_2", "p7_3","p7_4","p7_5","p7_6","p7_7"]) Then
 {
    switch (getPlayerUID(player)) do
```
Comment (ArwynFr) [inline initPlayerLocal.sqf]: Are the UIDs up to date ?

**Comment by ArwynFr** [comment]
why not put /normandie/ directory in /functions/ ?

**Comment by ImplicitDeny** [comment]
> 
> 
> why not put /normandie/ directory in /functions/ ?

Normandie scripts have no chance to run more than once. It is not relevant to build them as a function : no performance gain will ever be visible. No need to load `description.ext` with more functions.

**Comment by ArwynFr** [comment]
It's not about rewriting them as functions by to store them in a directory with all 3nd-party code.
Could be a /lib/ directory if you prefer to separate functions from non functions.

**Comment by ImplicitDeny** [comment]
>Could be a /lib/ directory if you prefer

Such a directory will only contain the `normandie` stuff. EOS must be at mission root, preslots will be rewritten and built as functions, and airdrop must be at mission root too.
This is a behaviour stabiolization update, not a full rewrite ;)

**Comment by ArwynFr** [comment]
> This is a behaviour stabiolization update, not a full rewrite ;)

Meh

File: initPlayerLocal.sqf (hunk 3 above)
Comment (ImplicitDeny) [reply initPlayerLocal.sqf]: ID are legacy ones. But the fact is p7, p7_1, etc, are nowhere in the mission file.

And by the way, it is not possible to design reserved slots by such a script. Run this script implies that the player has been able to connect, but if the server is full, he cannot even pass the lobby.

**Pull request merged**